RELEASE.md

```diff
@@ -0,0 +1,52 @@
+# How to make a release
+
+`chartpress` is a package [available on
+PyPI](https://pypi.org/project/chartpress/). These are instructions on how to
+make a release on PyPI.
+
+For you to follow along according to these instructions, you need:
+- To be a maintainer of the [PyPI chartpress
+  project](https://pypi.org/project/chartpress/).
+- To have push rights to the chartpress GitHub repository.
+
+## Technical steps to make a release
+
+1. Checkout master and make sure your up to date.
+
+   ```
+   git checkout master
+   git fetch <upstream> master
+   git reset --hard <upstream>/master
+   ```
+
+1. Update [CHANGELOG.md](CHANGELOG.md) and verify [README.md](README.md) has an
+   updated output of running `--help`.
+
+1. Set the `__version__` variable in [chartpress.py](chartpress.py)
+   appropriately and make a commit with message `release <tag>`.
+
+1. Create a git tag for the commit.
+
+   ```
+   git tag -a <tag> -m <tag> HEAD
+   ```
+
+1. Package the release
+   ```
+   python3 setup.py bdist_wheel
+   ```
+
+1. Upload it to PyPI
+   ```
+   twine upload dist/chartpress-<tag>-py3-none-any.whl
+   ```
+
+1. Reset the `__version__` variable in [chartpress.py](chartpress.py)
+   appropriately with a `.dev` appendix and make a commit with the message `back
+   to dev`.
+
+1. Push your two commits and tag to master.
+
+   ```
+   git push --tags <upstream>
```

Comment:
Is this definitely correct?
```
$ git push --help
...
       --tags
           All refs under refs/tags are pushed, in addition to refspecs
           explicitly listed on the command line.
...
```
suggests that won't push master, though it might just be the documentation is unclear.

Reply:
I think it'll push them to upstream repo, rather than the master branch there.

To do both, git push --tags upstream master

Reply:
Actually I was wrong.

git push --tags upstream will actually push to master but also to every other matching branch! See https://makandracards.com/makandra/8039-git-how-to-configure-git-to-push-only-your-current-branch. This might push a bunch of branches you didn't intend.

I woul still recommend explicitly setting name of branch to push

Reply:
Nice catch!

Reply:
Hmmm after reading this, I realize that created tags only relate to commits and not branches:

https://stackoverflow.com/questions/14613540/do-git-tags-only-apply-to-the-current-branch/27154277#27154277

And I believe that `git push --tags <remote> <branch>` would push the branch along with tags. I googled about pushing a single tag and found [this stackoverflow question with two important answers](https://stackoverflow.com/questions/5195859/how-do-you-push-a-tag-to-a-remote-repository-using-git).

To push a single tag:

```
git push <remote> <tag>
# or the same done more explicitly
git push <remote> tag <tag>
```

To push all annotated tags that are referencing a commit on branch

```
git push --follow-tags
```